conformance/ion_encoding/trivial_forms.ion

```diff
@@ -0,0 +1,26 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+// Non-sexps pass through
```

Comment:
Now that we have `$ion_literal`, should nonsensical things like `$ion_encoding::null` be errors instead?